src/main/java/seedu/duke/lesson/LessonList.java

```diff
@@ -1,4 +1,36 @@
 package seedu.duke.lesson;
 
+import java.util.ArrayList;
+
 public class LessonList {
-}
+    private ArrayList<Lesson> lessonList = new ArrayList<>();

```

Comment:
Consider using `List<Lesson> lessonList = new ArrayList<>()` instead to improve flexibility.

Note: [Relevant info](https://stackoverflow.com/questions/2279030/type-list-vs-type-arraylist-in-java)

Reply:
Hmm.. actually might be better to just declare it first, without initializing. Initialization should be done inside the constructor.